packages/config/config/devices/0x0060/an157.json

```diff
@@ -1,22 +0,0 @@
-// Everspring AN157
```

Comment:
And here?

Reply:
Not sure but I re-added it. Had to have been my manual mistake as this runs non-destructively (at least for now).